lib/streamlit/elements/form.py

```diff
@@ -246,12 +251,17 @@ def form_submit_button(
             True if the button was clicked.
         """
         ctx = get_script_run_ctx()
+        type = streamlit.elements.utils.check_valid_button_type(
+            str(type), "st.form_submit_button"
```

Comment:
Feedback appreciated as I'm not sure this is the proper way to import/call the `check_valid_button_type` function 🤔 

In `button.py` I can import more like:
```python
from streamlit.elements.utils import check_valid_button_type
```
but doing so here causes a circular import error in `form.py` file (since the utils file imports the `is_in_form` function).